### PR TITLE
AAP-31447: Marketing report

### PR DESCRIPTION
**Pull request by goneri**
Move the "Organization name" column and remove the "Trial expired_at" column.

### CODE REVIEW

File: ansible_ai_connect/users/reports/tests/test_generators.py [hunk 1]
```diff
@@ -209,10 +209,7 @@ def tearDown(self):
         super().cleanup()
 
     def get_report_header(self) -> str:
-        return (
-            "OrgId,UUID,First name,Last name,Email,Organization name,"
-            "Plan name,Trial started,Trial expired_at"
-        )
+        return "OrgId,UUID,First name,Last name,Organization name,Email," "Plan name,Trial started"
```
Comment (manstis) [inline ansible_ai_connect/users/reports/tests/test_generators.py:212]: You can concatenate this into a single string.

I suspect re-formatting removed the tuple but didn't join the strings.

**Review by manstis — DISMISSED** [review]
Minor comment, but otherwise LGTM 👍

File: ansible_ai_connect/users/reports/tests/test_generators.py (hunk 1 above)
Comment (goneri) [reply ansible_ai_connect/users/reports/tests/test_generators.py:212]: Unfortunately the line becomes too long and Flake8 will insult me.

**Review by manstis — APPROVED** [review]
LGTM 👍

**Pull request merged**